main.js

```diff
@@ -143,25 +158,37 @@ app = {
 	},
 
 	attack: () => {
-		// playerToAttack is an int, the index of the player list
-		const targetName = $("#player-list").children("tr").eq(config.playerToAttack)[0].innerText;
-		log(`. Now attacking ${targetName}`);
-		// click it, and then hack, and then a random port
-		$("#player-list").children("tr").eq(config.playerToAttack)[0].click();
-		$("#window-other-button").click();
-		const portNumber = getRandomInt(1,3);
-		// do a check for money
-		const portStyle = $(`#window-other-port${portNumber}`).attr("style");
-		if (portStyle.indexOf("opacity: 1") === -1) {
+		
+		// if the auto target is toggled, choose the target.
+		if (config.autotarget)
```

Comment:
If `config.autoTarget` is disabled, it continue attacking ports if `config.autoAttack` is enabled. It allow the player to stay on the same target to siphon it's BTC :) (sometimes you can make 1m+ per hack).